src/libANGLE/renderer/d3d/d3d11/winrt/CoreWindowNativeWindow.cpp

```diff
@@ -209,19 +209,19 @@ HRESULT GetCoreWindowSizeInPixels(const ComPtr<ABI::Windows::UI::Core::ICoreWind
 
 static float GetLogicalDpi()
 {
-    ComPtr<ABI::Windows::Graphics::Display::IDisplayPropertiesStatics> displayProperties;
-
-    if (SUCCEEDED(GetActivationFactory(HStringReference(RuntimeClass_Windows_Graphics_Display_DisplayProperties).Get(), displayProperties.GetAddressOf())))
-    {
-        float dpi = 96.0f;
-        if (SUCCEEDED(displayProperties->get_LogicalDpi(&dpi)))
-        {
-            return dpi;
-        }
-    }
-
-    // Return 96 dpi as a default if display properties cannot be obtained.
-    return 96.0f;
+	ComPtr<ABI::Windows::Graphics::Display::IDisplayInformation> displayInformation;
```

Comment:
This update doesn't appear to work - the real fix isn't terribly different though:

https://github.com/FNA-XNA/angle/commit/6119e24fabae7bc5269a8bd4a0fff025017d93d0